README.md

```diff
@@ -187,3 +188,21 @@ the car manually using the controller, use the following controls:
 
 The car starts in START mode. When pressed it will switch to MANUAL mode. Every
 subsequent press toggles between AUTO and MANUAL mode.
+
+
+Testing
+-------
+
+Testing is done with pytest. To run tests, make sure that the environment is
+initialized (run `rosvenv`, `rosinit`, and `rosdevel`) then run `pytest
+buzzmobile/tests`. This will run all tests in the tests directory. Be careful,
+beacuse there is a virtualenv in the root directory of the project, **running
+`pytest` will attempt to run thousands of unittests included with the python
+interpreter.**
+
+To write additional unit tests, please place within `buzzmobile/tests/unit` in
+directories that match the source files, that is tests for the
+`buzzmobile/process/gps_mapper` node should go in
+`buzzmobile/tests/unit/process/test_gps_mapper.py`. Integration and simulation 
+tests should go in the `integration` and `simulation` test subdirectories 
```

Comment:
nit: change to `buzzmobile/tests/integration` and `buzzmobile/tests/simulation` to avoid confusion